onnx/backend/test/case/node/maxpool.py

```diff
@@ -487,6 +487,44 @@ def export_maxpool_2d_ceil() -> None:
 
         expect(node, inputs=[x], outputs=[y], name="test_maxpool_2d_ceil")
 
+    @staticmethod
+    def export_maxpool_2d_ceil_output_size_reduce_by_one() -> None:
+        """input_shape: [1, 1, 2, 2]
+        output_shape: [1, 1, 1, 1]
+        """
+        node = onnx.helper.make_node(
+            "MaxPool",
+            inputs=["x"],
+            outputs=["y"],
+            kernel_shape=[1, 1],
+            strides=[2, 2],
+            ceil_mode=True,
+        )
+        x = np.array(
+            [
+                [
+                    [
+                        [
+                            1,
+                            2,
+                        ],
+                        [
+                            3,
+                            4,
+                        ],
+                    ]
+                ]
+            ]
```

Comment:
nit: remove the trailing comma to format this tightly. Do:

```suggestion
            [
                [
                    [
                        [
                            1,
                            2
                        ],
                        [
                            3,
                            4
                        ]
                    ]
                ]
            ]
```

then reformat again